From d49b4067529463fda1a25b7d3c1a365d3f323113 Mon Sep 17 00:00:00 2001
From: Thierry Fournier <tfournier@arpalert.org>
Date: Wed, 24 Feb 2016 08:13:13 +0100
Subject: [PATCH 51/67] BUG/MINOR: server: fix the format of the warning on
 address change

When the server address is changed, a message with unrequired '\n' or
'.' is displayed, like this:

   [WARNING] 054/101137 (3229) : zzzz/s3 changed its IP from 127.0.0.1 to ::55 by stats command
   .

This patch remove the '\n' which is sent before the '.'.

This patch must be backported in 1.6
(cherry picked from commit c62df8463b15ca0bff79309b94351480e302c59f)
---
 src/server.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/server.c b/src/server.c
index 874ddf2..c76a469 100644
--- a/src/server.c
+++ b/src/server.c
@@ -819,11 +819,11 @@ const char *server_parse_addr_change_request(struct server *sv,
 	unsigned char ip[INET6_ADDRSTRLEN];
 
 	if (inet_pton(AF_INET6, addr_str, ip)) {
-		update_server_addr(sv, ip, AF_INET6, "stats command\n");
+		update_server_addr(sv, ip, AF_INET6, "stats command");
 		return NULL;
 	}
 	if (inet_pton(AF_INET, addr_str, ip)) {
-		update_server_addr(sv, ip, AF_INET, "stats command\n");
+		update_server_addr(sv, ip, AF_INET, "stats command");
 		return NULL;
 	}
 
-- 
2.4.10

